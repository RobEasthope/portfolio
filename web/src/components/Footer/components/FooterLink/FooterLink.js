import styled from 'styled-components';
import { Link } from 'rebass';

import { colors } from '../../../../styles/theme';
import fontSizing from '../../../../utils/fontsizing';

const FooterLink = styled(Link)`
  font-size: ${fontSizing(18, 24)};
  color: ${colors.ink};
`;

export default FooterLink;
