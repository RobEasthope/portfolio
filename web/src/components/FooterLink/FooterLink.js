import styled from 'styled-components';
import { Link } from 'rebass';

import palette from '../../styles/palette';
import fontSizing from '../../utils/fontsizing';

const FooterLink = styled(Link)`
  font-size: ${fontSizing(18, 24)};
  color: ${palette.ink};
`;

export default FooterLink;
