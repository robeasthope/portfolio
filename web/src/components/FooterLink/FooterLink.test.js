import React from 'react';
import renderer from 'react-test-renderer';

import FooterLink from './FooterLink';

describe('FooterLink', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<FooterLink href="https://www.google.com">Google.com</FooterLink>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
