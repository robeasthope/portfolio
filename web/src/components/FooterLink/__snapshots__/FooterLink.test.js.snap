// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterLink renders correctly 1`] = `
<a
  className="sc-bdVaJa sc-ifAKCX sc-htoDjs cPtkgc"
  color="blue"
  href="https://www.google.com"
>
  Google.com
</a>
`;
